tests/test_lldpSyncDaemon.py

```diff
@@ -123,11 +134,11 @@ def test_loc_chassis(self):
         self.daemon.sync(parsed_update)
         db = create_dbconnector()
         db_loc_chassis_data = db.get_all(db.APPL_DB, 'LLDP_LOC_CHASSIS')
-        self.assertEquals(parsed_loc_chassis, db_loc_chassis_data)
+        self.assertEqual(parsed_loc_chassis, decode_dict(db_loc_chassis_data))
 
     def test_remote_sys_capability_list(self):
         interface_list = self._interface_only['lldp'].get('interface')
         for interface in interface_list:
-            (if_name, if_attributes), = interface.items()
+            (if_name, if_attributes), = list(interface.items())
```

Comment:
I believe this `list()` conversion is unnecessary

Reply:
@jleveque Updated